### PR TITLE
Jira DOC 910: RC - Update "View/edit databases" to match recent workflow changes.

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A stages preview of this work [is available](https://docs.redis.com/staging/jira-doc-910/rc/databases/view-edit-database/).

File: content/rc/databases/view-edit-database.md [hunk 1]
```diff
@@ -9,24 +9,25 @@ aliases: /rc/administration/setup-and-editing/viewing-editing-database/
...
     {{<image filename="images/rc/database-details-configuration-tab-general-flexible.png" width="75%" alt="The Configuration tab of the Database details screen." >}}{{< /image >}}
 
 The **Database** screen lets you review:
 - Configuration details of a database
 - Graphs showing performance metrics
-- Recent activity via a "slowlog"
+- Recent activity via a "slowlog," which lists queries that exceed a certain [execution time](https://redis.io/commands/slowlog).
 
-To change database settings, see [Edit database details](#edit-database-details),
+For help changing database settings, see [Edit database details](#edit-database-details),
```
Comment (rrelledge) [inline content/rc/databases/view-edit-database.md:30]:
```suggestion
For help changing database settings, see [Edit database details](#edit-database-details).
```

File: content/rc/databases/view-edit-database.md [hunk 2]
```diff
@@ -145,6 +146,32 @@ Here, you can:
 
 For best results, we recommend backing up data before starting any **Danger Zone** actions.
 
+## Manage the database list
+
+The **Databases** list summarizes the status of all databases from the subscriptions associated with your account.  
+
+You can:
+
+- Sort the list in descending or ascending order using the the arrow displayed to right of the field name in the header.  Supported fields include _Subscription_, _Name_, and _Memory_.
```
Comment (rrelledge) [inline content/rc/databases/view-edit-database.md:155]: Do you want these italicized fields to be bold instead to match the formatting in the next bullet point?
```suggestion
- Sort the list in descending or ascending order using the the arrow displayed to right of the field name in the header.  Supported fields include **Subscription**, **Name**, and **Memory**.
```

**Review by rrelledge — APPROVED** [review]
Looks good to me.

**Review by kaitlynmichael — APPROVED** [review]
Looks great!

File: content/rc/databases/view-edit-database.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/databases/view-edit-database.md]: Fixed.

File: content/rc/databases/view-edit-database.md (hunk 2 above)
Comment (lanceleonard) [reply content/rc/databases/view-edit-database.md]: Fixed.

**Pull request merged**